js/RateCurve.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
```

Comment:
That was the Idea, since it's unlikely someone is going to use it elsewhere, an additional dependency won't hurt.
Ok, I'll introduce a function parameter into constructor and pass it from receiver and pid tuning tabs.